portal/src/main/webapp/WEB-INF/jsp/patient_view/patient_view.jsp

```diff
@@ -475,13 +475,15 @@ function initOncoKB(instanceId, ids, mapData, type, indicatorCallback) {
                     indicatorCallback(instance);
                 }
             });
-            instance.getCivicIndicator().then(function () {
-                if(_.isFunction(indicatorCallback)) {
-                    indicatorCallback(instance);
-                }
-            }).fail(function() {
-                console.log('getCivicIndicator failed.');
-            });
+            if (showCivic) {
```

Comment:
This works. But getCivicIndicator itself already have `if (showCivic)` condition check. I would rather reuse that. In order to do so, you probably want to return a promise inside of `getCivicIndicator` and return def.reject() if the CIViC is disabled.